td-agent/Rakefile

```diff
@@ -494,6 +561,9 @@ class BuildTask
 
       desc "Create configuration files for Windows"
       task :msi_config => [:td_agent_config, :wix_config, :win_batch_files]
+
+      desc "Create configuration files for macOS"
+      task :pkg_config => [:td_agent_config, :pkgbuild_scripts, :pkgbuild_config, :bin_scripts, :launchctl_config]
```

Comment:
It may be better to use pkgbuild_config or something. (dmg_config? to avoid confusing with `pkg-config` command)

Reply:
dmg_config would be better for me.